index.bs

```diff
@@ -486,6 +486,69 @@ reasonable future change. We could run [=process a network cookie access for bou
 in the <a spec="fetch">HTTP-network-or-cache fetch</a> algorithm after step 8.21.1.2,
 "... [=append=] (`Cookie`, cookies) to httpRequest’s [=header list=]. ..."
 
+<h5 id="bounce-tracking-mitigations-service-worker-activation-monkey-patch">Service Worker Activation Monkey Patch</h5>
+
+Each [=top-level traversable=] maintains a record of which sites have activated service workers in the current [=extended navigation=].
+
+Insert the following steps in the [activate]((https://w3c.github.io/ServiceWorker/#activate) algorithm before step 11,
+"Run the [Update Worker State](https://w3c.github.io/ServiceWorker/#update-worker-state) algorithm...":
+
+1. Run [=process a service worker activation for bounce tracking mitigations=] given <var ignore>registration</var> and <var ignore>matchedClients</var>.
```

Comment:
I don't think this is the right place.  Updating the worker state is somewhat disconnected from a navigation itself because it can happen in the background.

I was more worried about the navigation path itself.  So if we end up hitting the Run Service Worker algorithm in step 23 for a non-subresource request here:

https://w3c.github.io/ServiceWorker/#on-fetch-request-algorithm

All other accesses of storage in ServiceWorker should in theory be caught by monitoring the "obtain a storage bottle map" with an environment settings object hook.